app/views/volunteers/_form.html.slim

```diff
@@ -1,119 +1,114 @@
-= simple_form_for(@volunteer) do |f|
-  - if f.error_notification.present?
-    .row
-      .col-xs-12
-        = f.error_notification
-
+- if f.error_notification.present?
   .row
-    .col-xs-12.col-md-6
-        = f.input :first_name
+    .col-xs-12
+      = f.error_notification
 
-    .col-xs-12.col-md-6
-        = f.input :last_name
+.row
+  .col-xs-12.col-md-6
+      = f.input :first_name
 
-  .row
-    .col-xs-12.col-md-6
-      = f.input :date_of_birth, as: :date, start_year: Date.today.year - 90,
-        end_year: Date.today.year, include_blank: true
+  .col-xs-12.col-md-6
+      = f.input :last_name
 
-    .col-xs-12.col-md-6
-      = f.input :gender, collection: gender_collection, as: :radio_buttons
+.row
+  .col-xs-12.col-md-6
+    = f.input :date_of_birth, as: :date, start_year: Date.today.year - 90,
+      end_year: Date.today.year, include_blank: true
 
-  .row
-    .col-xs-12
-      = f.input :avatar, as: :file
+  .col-xs-12.col-md-6
+    = f.input :gender, collection: gender_collection, as: :radio_buttons
 
-  .row
-    .col-xs-12.col-md-6
-      = f.input :nationality, as: :country
-    .col-xs-12.col-md-6
-      = f.input :additional_nationality, as: :country
+.row
+  .col-xs-12
+    = f.input :avatar, as: :file
 
-  .row
-    .col-xs-12
-      = f.input :street
+.row
+  .col-xs-12.col-md-6
+    = f.input :nationality, as: :country
+  .col-xs-12.col-md-6
+    = f.input :additional_nationality, as: :country
 
-    .col-xs-6
-      = f.input :zip
+.row
+  .col-xs-12
+    = f.input :street
 
-    .col-xs-6
-      = f.input :city
+  .col-xs-6
+    = f.input :zip
 
-    .col-xs-12.col-md-6
-      = f.input :email
+  .col-xs-6
+    = f.input :city
 
-    .col-xs-12.col-md-6
-      = f.input :phone
+  .col-xs-12.col-md-6
+    = f.input :email
 
-  .row
-    .col-xs-12
-      = f.input :profession
-      = f.input :education
-      = f.input :motivation, label: t('your_motivation')
-      = f.input :experience, label: t('your_experience')
-      = f.input :expectations, label: t('your_expectations')
-      = f.input :strengths, label: t('your_strengths')
-      = f.input :skills, label: t('your_skills')
-      = f.input :interests, label: t('your_interests')
+  .col-xs-12.col-md-6
+    = f.input :phone
 
-  .row
-    .col-xs-12
-      = f.input :state
+.row
+  .col-xs-12
+    = f.input :profession
+    = f.input :education
+    = f.input :motivation, label: t('your_motivation')
+    = f.input :experience, label: t('your_experience')
+    = f.input :expectations, label: t('your_expectations')
+    = f.input :strengths, label: t('your_strengths')
+    = f.input :skills, label: t('your_skills')
+    = f.input :interests, label: t('your_interests')
 
   .row
     .col-xs-12
-      = f.input :duration, collection: Volunteer.duration_collection, as: :radio_buttons
+      = f.input :state, as: :hidden
```

Comment:
shouldn't it be not displayed to the volunteers themself. So thru `policy(Volunteer).state_field?` or so?